moto/ssm/models.py

```diff
@@ -1286,6 +1287,7 @@ def create_document(
         self._documents[ssm_document.name] = documents
 
         if tags:
+            # pytest.set_trace()
```

Comment:
Same here (and the `pytest`-import)

Reply:
hi, i've removed this comment in my latest commit